lib/zipkin-tracer/zipkin_json_tracer.rb

```diff
@@ -21,8 +22,15 @@ module Trace
   # This class sends information to the Zipkin API.
   # The API accepts a JSON representation of a list of spans
   class ZipkinJsonTracer < ZipkinTracerBase
+
+    def initialize(options)
+      SuckerPunch.logger = options[:logger]
+      @json_api_host = options[:json_api_host]
+      super(options)
+    end
+
     def flush!
-      AsyncJsonApiClient.new.async.perform(@options[:json_api_host], @spans.values.dup)
+      AsyncJsonApiClient.new.async.perform(@json_api_host, spans.values.dup)
```

Comment:
IMHO attr accessors become the public API . Specially being a gem we do not know who may be accesing it for what reasons. If possible I want to keep things private.